### PR TITLE
FIX: Jobs not triggering after restart/errors

### PR DESCRIPTION
**Pull request by jacquesvcritien**

### CODE REVIEW

File: api/src/flux.js [hunk 1]
```diff
@@ -60,8 +60,8 @@ export const makeFluxNotifier = async (
   let currentRound;
 
   // Start a fresh query if we don't already have one.
-  const triggerQuery = async () => {
-    if (!querying) {
+  const triggerQuery = async (fromPollTimer) => {
+    if (!querying or fromPollTimer) {
```
Comment (michaelfig) [inline api/src/flux.js:64]: This seems to be a syntax error.

```suggestion
    if (!querying || fromPollTimer) {
```

**Review by michaelfig — CHANGES_REQUESTED** [review]
I'm not sure if you have tested this code.  The logic looks good though.

Can you fix the syntax error and try it out before re-requesting a review?

File: api/src/flux.js (hunk 1 above)
Comment (jacquesvcritien) [reply api/src/flux.js:64]: Ah yes sorry, this is because I initiated the PR from Github. Sorry about that, the rest of the code should be fine

**Review by michaelfig — APPROVED** [review]
Great stuff!

**Pull request merged**